examples/grpc-bidirectional-streaming-copy/grpc_bidirectional_streaming.out

```diff
@@ -0,0 +1,4 @@
+$ bal grpc --input grpc_bidirectional_streaming.proto  --output stubs
+Continuing with the existing protoc executor file at /var/folders/fr/95k6869j5yd_nrx04qztpmdr0000gp/T/protoc-3.9.1-osx-x86_64.exe
```

Comment:
```suggestion
```

Reply:
shall we delete this entry? 
this line varies depending on OS

Reply:
Done.